src/structures/vroom/cost_wrapper.h

```diff
@@ -16,12 +16,23 @@ All rights reserved (see LICENSE).
 namespace vroom {
 
 struct CostWrapper {
+  const static uint32_t DIVISOR = 100;
+
   const double durations_factor;
+  const uint32_t discrete_factor;
   const Matrix<Cost>* durations_matrix;
```

Comment:
It looks like we can get rid of the `durations_matrix` pointer now since we access `cost_data` directly.

Reply:
Done, I removed `durations_factor`as well.